src/storage/src/region/tests.rs

```diff
@@ -67,6 +67,10 @@ impl<S: LogStore> TesterBase<S> {
         }
     }
 
+    pub async fn close(&self) {
```

Comment:
Maybe we could add a close() method to the region trait later. You could create an issue for this (I'm not sure if there is already an issue for similar problems.

Reply:
> Maybe we could add a close() method to the region trait later. You could create an issue for this (I'm not sure if there is already an issue for similar problems.

I created an issue #832 for this.